src/types/types.d.ts

```diff
@@ -141,3 +147,27 @@ interface APITweet {
 
   twitter_card: 'tweet' | 'summary' | 'summary_large_image' | 'player';
 }
+
+interface APIUser {
+  id: string;
+  name: string;
+  screen_name: string;
+  avatar_url: string;
+  banner_url: string;
+  avatar_color: string;
+  description: string;
+  location: string;
+  url: string;
+  protected: boolean;
+  verified: boolean;
```

Comment:
Maybe we could pass through the type of verification instead of just verified

(i.e. Business, Twitter Blue, Legacy)

Reply:
API idea:

verified: 'legacy' | 'blue' | 'business' | 'government' | null

For government in particular we could also perhaps pass through the label Twitter gives accounts (i.e. Canadian government organization, United States government official, Russia state-affiliated media)